File: connect2016/developer/03/dotnet/workshop-dotnet/workshop-dotnet/Models/Person.cs
using Newtonsoft.Json;

namespace workshop_dotnet.Models
{
    public class Person
    {
        [JsonProperty("document_id")]
        public string Id { get; set; }

        [JsonProperty("firstname")]
        public string FirstName { get; set; }

        [JsonProperty("lastname")]
        public string LastName { get; set; }

        [JsonProperty("email")]
        public string Email { get; set; }
    }
}

File: connect2016/developer/03/java/app/src/main/java/com/couchbase/workshopapp/entity/Person.java
package com.couchbase.workshopapp.entity;

import com.couchbase.client.java.repository.annotation.Id;
import org.springframework.data.annotation.Version;

public class Person {
	@Id
	public String id;

	@Version
	public long Version;

	public String firstName;

	public String lastName;

	public String email;

	@Override
	public String toString(){
		return  "firstName:" + this.firstName + "," +
				"lastName:" + this.lastName + "," +
				"email:" + this.email + "," +
				"cas:" + this.Version;
	}

}